src/homepageExperience/components/steps/cli/ExecuteAggregateQuery.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import CodeSnippet from 'src/shared/components/CodeSnippet'
+
+import {SafeBlankLink} from 'src/utils/SafeBlankLink'
+import {event} from 'src/cloud/utils/reporting'
+
+const logCopyCodeSnippet = () => {
+  event('firstMile.cliWizard.executeAggregateQuery.code.copied')
+}
+
+const logDocsOpened = () => {
+  event('firstMile.cliWizard.executeAggregateQuery.docs.opened')
+}
+
+type OwnProps = {
+  bucket: string
+}
+
+export const ExecuteAggregateQuery = (props: OwnProps) => {
+  const {bucket} = props
+
+  const fromBucketSnippet = `from(bucket: "weather-data")
+  |> range(start: -10m)
+  |> filter(fn: (r) => r.measurement == "temperature")
+  |> mean()`
+
+  const codeSnippet = `influx query 'from(bucket:"${bucket}") |> range(start:-10m) |> mean()' --raw`
+
+  return (
+    <>
+      <h1>Execute a Flux Aggregate Query</h1>
+      <p>
+        An{' '}
+        <SafeBlankLink
+          href="https://docs.influxdata.com/flux/v0.x/function-types/#aggregates"
+          onClick={logDocsOpened}
+        >
+          aggregate
+        </SafeBlankLink>{' '}
+        function is a powerful method for returning combined, summarized data
+        about a set of time-series data.
```

Comment:
@Luckette @taramk the use of `summarized data` and `time-series data` in the same sentence reads a little bit awkward and seems to be different than the [docs description of an aggregate function](https://docs.influxdata.com/influxdb/cloud/query-data/flux/window-aggregate/). "[Aggregate functions](https://docs.influxdata.com/flux/v0.x/function-types#aggregates) take the values of all rows in a table and use them to perform an aggregate operation. The result is output as a new value in a single-row table."

Reply:
thanks, bucky. I'll make sure to fix the wording once a ticket is created.